performant-software/performant-web
# Removed green badge from contact page and format badge for collections

## What does this PR do?
- Removed badge from the contact page
- Create custom SCSS file to overwrite badge colors from the "front" theme
- Documented SCSS  
- This closes #63

## How to test?
- Navigate to contact, this page should not have a badge
- Navigate to "services/any-service", "projects/any-project". Pages should have a badge link to the collection index page
